packages/react-composites/src/composites/CallComposite/utils/Utils.ts

```diff
@@ -542,3 +542,15 @@ export const getBackgroundEffectFromSelectedEffect = (
  */
 export const getSelectedCameraFromAdapterState = (state: CallAdapterState): VideoDeviceInfo | undefined =>
   state.devices.selectedCamera || state.devices.cameras[0];
+
+/**
+ * Helper to determine if the adapter has a locator or targetCallees
+ * @param locatorOrTargetCallees
+ * @returns boolean to determine if the adapter has a locator or targetCallees, true is locator, false is targetCallees
+ * @private
+ */
+export const getLocatorOrTargetCallees = (
+  locatorOrTargetCallees: CallAdapterLocator | StartCallIdentifier[]
+): locatorOrTargetCallees is CallAdapterLocator => {
+  return !!(('groupId' || 'meetingLink' || 'roomId') in locatorOrTargetCallees);
```

Comment:
does `('groupId' || 'meetingLink' || 'roomId')` work? My assumption here is that javascript will do whats in the brackets first and compute `('groupId' || 'meetingLink' || 'roomId')` to be `true` and therefore test if `!!(true) in locatorOrTargetCallees`

Reply:
I also worry that we don't capture every possible locator by checking each one of these (including future ones). Does it work to flip and check if its an array instead and see if its a TargetCallees?

Reply:
yup I can flip the logic, would also remove the need for it to be a check like this. I think we can still look for the array methods and that would give us the right answer here no? (do we think they might ever make a locator that is just an array?)